AI Game Jam/Assets/Scripts/PlayerMovement.cs

```diff
@@ -1,33 +1,52 @@
+/*
+* Description: Manages the Player Movement, uses the built in character controller component in unity
+* Author: Chase Bennett-Hill
+* Last Modified: 7 / 03 / 24
+* Last Modified By: Chase Bennett-Hill
+*/
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour
 {
-    private CharacterController controller;
-    private Vector3 playerVelocity;
-    private bool groundedPlayer;
-    [SerializeField] private float playerSpeed = 4.0f;
+
+    private CharacterController controller; //Character controller component
+    private Vector3 playerVelocity; //Player velocity value based on x y and z axis
+    [SerializeField] private bool groundedPlayer; //Checks if the player is touching the ground
+    [SerializeField] private float playerSpeed = 4.0f; //Player speed
+    [SerializeField] private float jumpHeight = 1.0f; //Jump height
+    private const float GRAVITYVALUE = -9.81f;
```

Comment:
I changed this to 20 or 25 and it makes for a much stronger pull to the ground--feels a lot more natural. This is also important for the game as this current value, the player can leap off of a cube and it feels like they are gliding off which may be game-breaking for the future but mostly doesn't feel natural or what should happen in reality--the player should just plummet to the ground when they fall.

Reply:
I will have a look